build/spot.js

```diff
@@ -2725,6 +2725,86 @@ function addPhiWrapped(candidates, options){
 }
 
 })();
+
+function containsClitic(x){
+	return x.indexOf("clitic")>-1;
+}
+
+function generateWordOrders(wordList, clitic){
+	if(typeof wordList === 'string'){
+		var cliticTagIndex = wordList.indexOf("-clitic");
+		if(cliticTagIndex > 0){
+			var wordListParts = wordList.split("-clitic");
+			wordList = wordListParts[0]+wordListParts[1];
+		}
+		wordList = wordList.split(' ');
+	}
+	//Find the clitic to move around
+	var cliticIndex = wordList.indexOf(clitic);
+	if(cliticIndex < 0)
+		throw new Error("The provided clitic"+clitic+" was not found in the word list");
```

Comment:
Would be good to have a space between between the string "clitic" and the clitic.

Reply:
fixed